Makefile

```diff
@@ -45,7 +45,7 @@ release:
 	$(call build,tcld,windows,amd64,.exe)
 
 tools:
-	@go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.51.2
+	@go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.60.1
```

Comment:
upgrading to `v1.60.1` fixes linting, since `1.51.2` is incompatible with the version of go installed by GH actions.